book/P3C15_svm.qmd

```diff
@@ -14,6 +14,9 @@ Support vector machines are a popular class of models in regression and classifi
 Survival support vector machines (SSVMs) predict continuous responses that can be used as ranking predictions with some formulations that provide survival time interpretations.
 This chapter starts with SVMs in the regression setting before moving to adaptions for survival analysis.
 
+As of the time of publication, no SSVMs for competing risks appear to have been published [@Kantidakis2023; @Monterrubio-Gómez2024; @Djangang2025].
+Theoretically, one could use any of the methods to estimate per-cause risk by considering each risk separately and censoring observations that experience a different risk, however this has not been validated in the literature.
```

Comment:
the important part in the caus-specific paradigm is that the learner returns a hazard estimate. If thats the case, you can calculate the CIF. If it only returns the surv prob, one could try to approximate a hazard estimate from S(t) (e.g. via numeric differentiation and some assumptions), but it's less common. not sure if SVM provides hazard estimates